src/providers/packet/mock_tests.rs

```diff
@@ -53,7 +53,44 @@ fn test_packet_attributes() {
         "ssh_keys": [],
         "network": {
             "interfaces": [],
-            "addresses": [],
+            "addresses": [
+              {
+                "id": "fde74ec8-bc24-43ca-a852-875bd6e10bee",
+                "address_family": 4,
+                "netmask": "255.255.255.254",
+                "public": true,
+                "management": true,
```

Comment:
Out of curiosity, what does this field mean?

Reply:
AFAIK a "management IP" is an IP address that is bound to a Packet device (server). Packet offers also "Elastic IPs", customer-provided IPs etc.
Here, for example, is what what an Elastic IP (which is a non-management IP) looks like when attached to a device:

```json
{
  "id": "d392685e-072c-419b-a588-402b48d77bdc",
  "address_family": 4,
  "netmask": "255.255.255.255",
  "created_at": "2020-02-18T09:27:51Z",
  "public": true, 
  "cidr": 32,
  "management": false,
  "enabled": true, 
  "network": "147.1.1.1",
  "address": "147.1.1.1",
  "gateway": "147.1.1.1",
  "parent_block": {
    "network": "147.1.1.1",
    "netmask": "255.255.255.255",
    "cidr": 32,
    "href": "/ips/fdaa9c07-3dbc-47fd-8970-c3161888136b"
  }
}
```